cypher-shell/src/integration-test/java/org/neo4j/shell/commands/CypherShellVerboseIntegrationTest.java

```diff
@@ -207,4 +207,25 @@ public void paramsAndListVariablesWithSpecialCharacters() throws CommandExceptio
         assertThat(queryResult.get(0), containsString("\n| " + randomLong+ " |\n"));
         assertEquals(randomLong, shell.getAll().get("bob"));
     }
+
+    @Test
+    public void cypherWithOrder() throws CommandException {
+        // given
+        shell.execute( "CREATE INDEX ON :Person(age)" );
+
+        //when
+        shell.execute("CYPHER RUNTIME=INTERPRETED EXPLAIN MATCH (n:Person) WHERE n.age >= 18 RETURN n.name, n.age");
+
+        //then
+        ArgumentCaptor<String> captor = ArgumentCaptor.forClass(String.class);
+        verify(logger, times(2)).printOut(captor.capture());
+
+        List<String> result = captor.getAllValues();
+        String actual = result.get(0);
+        if ( actual.contains( "CYPHER 3.5" )) // Sadly best way to have test that relies on 3.5 functionality...
```

Comment:
This does mean this test will turn off for 3.6 and 4.x. Another option would be to pull out the x.y and check that the version is >= 3.5. Not totally reliable, as it makes assumptions about our version numbering, but they are assumptions that have remained valid for 10 years, so seem safe enough.